### DISCUSSION TITLE
Welcome to ext-remover Discussions!

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Loganrocks86** [comment]
chrome.google.com is blocked, so the webstore and webstorex is blocked. Is there a different link I can use the bookmarklet ext-remover in?

**Reply by 3kh0** [reply]
please start a new discussion for that
